ExomeDepth/check_gender_bam.py

```diff
@@ -4,39 +4,85 @@
 import pysam
 import settings
 
```

Comment:
Using Samtools will only add an additional dependency outside Python. If idxstats might be available using Pysam it could be a good option thought. However, as I said in the previous comment, we do not look at the entire chromosome, but a region. Moreover, we also perform a filtering on mapping quality (<20) which also would be present in a more general approach like idxstats (and thus increases noise)